app/javascript/components/TiptapExtensions/Link.tsx

```diff
@@ -250,7 +250,7 @@ const LinkNodeView = ({ node, editor, getPos, deleteNode }: NodeViewProps) => {
               onChange={(evt) => setLink({ ...link, url: evt.target.value })}
               onKeyDown={handleKeyPress}
             />
-            <div className="input-with-button">
+            <div className="flex items-end gap-2">
```

Comment:
Since these are two buttons and not actually an input and a button, using `items-end` makes this look a little better IMO
<img width="263" height="63" alt="image" src="https://github.com/user-attachments/assets/f120ecb5-7193-4c6d-9b3b-6a32684d8d63" />
<img width="263" height="63" alt="image" src="https://github.com/user-attachments/assets/f8611d96-8aec-4957-bda6-328d950e7c30" />

Reply:
(For future reference, would be easier to compare if the screenshots showed the rest of the form/popup)